api/src/test/java/com/foro/api/TestTopicRepositoryIT.java

```diff
@@ -0,0 +1,149 @@
+        Assert.assertEquals(topicToBeCompared.getTopicStatus(), topicWhoComparesTo.getTopicStatus());
+        Assert.assertEquals(topicToBeCompared.getIdUser(), topicWhoComparesTo.getIdUser());
+        Assert.assertEquals(topicToBeCompared.getCourse(), topicWhoComparesTo.getCourse());
+        }
+        @Test
+    public void testListTopics(){
+
+        String title = "What means Java";
+        String message = "I dont understand the difference between Java and Python";
+        String creationDateString = "2022/05/04 12:30:00";
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
+        //LocalDateTime.parse("2023-11-07T17:24:36.859762975");
+        LocalDateTime creationDate = LocalDateTime.parse(creationDateString, formatter);
+        TopicStatus topicStatus = TopicStatus.CLOSED;
+        Integer idUser = 4;
+        Course course = Course.JAVA;
+        Boolean deleteTopic = false;
+
+        TopicDTO topicInformation1 = TopicDTO.builder()
+                .title(title)
+                .message(message)
+                .creationDate(creationDate)
+                .topicStatus(topicStatus)
+                .idUser(idUser)
+                .course(course)
+                .deleteTopic(deleteTopic)
+                .build();
+
+        TopicDTO topic = topicRepository.save(topicInformation1);
+
+        String title2 = "Where could I comment in the foro page";
+        String message2 = "I dont know how to reply the questions in the page";
+        String creationDateString2 = "2023/05/05 11:30:00";
+        DateTimeFormatter formatter2 = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
+        //LocalDateTime.parse("2023-11-07T17:24:36.859762975");
+        LocalDateTime creationDate2 = LocalDateTime.parse(creationDateString2, formatter);
+        TopicStatus topicStatus2 = TopicStatus.CLOSED;
+        Integer idUser2 = 7;
+        Course course2 = Course.OTHER;
+        Boolean deleteTopic2 = false;
+
+        TopicDTO topicInformation2 = TopicDTO.builder()
+                .title(title2)
+                .message(message2)
+                .creationDate(creationDate2)
+                .topicStatus(topicStatus2)
+                .idUser(idUser2)
+                .course(course2)
+                .deleteTopic(deleteTopic2)
+                .build();
+
+        TopicDTO topic2 = topicRepository.save(topicInformation2);
+
+        List<TopicDTO> topics=topicRepository.findAll();
+        Assert.assertEquals(topics.size(),2);
+
+        topicRepository.deleteById(topic.getId());
+        topicRepository.deleteById(topic2.getId());
+
+        boolean idTopicDeleted=topicRepository.findById(topic.getId()).isPresent();
+        boolean idTopic2Deleted=topicRepository.findById(topic2.getId()).isPresent();
+
+        Assert.assertFalse(idTopicDeleted);
+        Assert.assertFalse(idTopic2Deleted);
```

Comment:
Let's use `isEmpty()` and `assertTrue(boolean)`, to do this, so it makes more sense when readed.

```java
        boolean idTopicDeleted=topicRepository.findById(topic.getId()).isEmpty();
        boolean idTopic2Deleted=topicRepository.findById(topic2.getId()).isEmpty();

        Assert.assertTrue(idTopicDeleted);
        Assert.assertTrue(idTopic2Deleted);
```